src/@types/MetaData.d.ts

```diff
@@ -20,6 +20,7 @@ export interface AdditionalInformationMarket extends AdditionalInformation {
   consent: {
     noPersonalData: boolean
   }
+  isInCompliance: boolean
```

Comment:
I think this is not a good name. compliant, already means something is complying, and already suggest a binary/boolean value. gxCompliant would make more sense. Or better:

```json
compliance: {
    gx: boolean
}
````

Reply:
Fixed!